assets/_scss/elements/_inputs.scss

```diff
@@ -78,6 +81,17 @@ select {
   background-position: right 1.3rem center;
 }
 
+legend {
```

Comment:
That was for the error input. Do you have another idea we could use?

Without it, it looks like this:

<img width="626" alt="screen shot 2015-09-08 at 4 06 27 pm" src="https://cloud.githubusercontent.com/assets/5249443/9749634/ac8cf0c8-5643-11e5-9894-183fb7222613.png">